src/components/notice/notice.tsx

```diff
@@ -106,15 +116,35 @@ export class Notice implements ConditionalSlotComponent {
    * Accessible name for the close button.
    *
    * @default "Close"
+   * @deprecated – translations are now built-in, if you need to override a string, please use `messageOverrides`.
    */
-  @Prop({ reflect: false }) intlClose: string = TEXT.close;
+  @Prop({ reflect: false }) intlClose: string;
 
   /** Specifies the size of the component. */
   @Prop({ reflect: true }) scale: Scale = "m";
 
   /** Specifies the width of the component. */
   @Prop({ reflect: true }) width: Width = "auto";
 
+  /**
+   * Made into a prop for testing purposes only
+   *
+   * @internal
+   */
+  @Prop({ mutable: true }) messages: Messages;
+
+  /**
+   * Use this property to override individual strings used by the component.
+   */
+  @Prop({ mutable: true }) messageOverrides: Partial<Messages>;
+
+  @Watch("intlClose")
+  @Watch("defaultMessages")
+  @Watch("messageOverrides")
+  onMessagesChange(): void {
+    /** referred in t9n util */
```

Comment:
Please use the same snippet from the t9n module doc. Reasoning: it'll be better if these are all consistent if we need a quick way to look them up.